src/css/base.scss

```diff
@@ -284,10 +288,14 @@ a:focus {
     top: 0;
 }
 
-.dc__left-35 {
```

Comment:
dc__left-35 seems to be used at some places. Please re-check & update accordingly.